LinusWKing/Teamwork-BackeEnd
# #29040698 Initial Package Tools Setup

### What does this PR do?
Setup and configure linting tools and ES6 features
#### Description of Task to be completed?
- Add require packages
- Configure the packages

#### How should this be manually tested?

- Clone the repository
- Start the server using npm run start
- Navigate to localhost:3000/api/v1 on browser
- You should see a message

#### Any background context you want to provide?
None

#### What are the relevant project board stories?
[#29040698](https://github.com/LinusWKing/Teamwork-BackeEnd/projects/1#card-29040698)

#### Screenshots (if appropriate)
None

#### Questions:
None
